tests/core/test_dataset.py

```diff
@@ -1393,3 +1393,11 @@ def test_active_t_coords_deprecated(mesh):
         mesh.active_t_coords = t_coords
         if pv._version.version_info[:2] > (0, 46):
             raise RuntimeError('Remove this deprecated property')
+
+
+def test_active_array_info_deprecated(mesh):
+    match = 'ActiveArrayInfo is deprecated. Use ActiveArrayInfoTuple instead.'
+    with pytest.warns(PyVistaDeprecationWarning, match=match):
+        pv.core.dataset.ActiveArrayInfo(association=pv.FieldAssociation.POINT, name='name')
+        if pv._version.version_info >= (0, 48):
```

Comment:
See #6850 and #6862.
```suggestion
        if pv._version.version_info[:2] > (0, 48):
```